.github/workflows/release_stable.yaml

```
@@ -32,9 +32,14 @@ jobs:
             echo "needs_releasing=false" >> $GITHUB_OUTPUT
           fi
 
-  release:
+  verify-image-id-onchain:
     needs: [check-if-needs-releasing]
     if: needs.check-if-needs-releasing.outputs.needs_releasing == 'true'
+    uses: ./.github/actions/verify-image-id-onchain
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/actions/verify-image-id-onchain": read /home/jailuser/git/.github/actions/verify-image-id-onchain: is a directory

(workflow-call)